City-of-Helsinki/hel-fi-drupal-grants
# UHF-10935: Added class and margin for delete attachment button

# [UHF-10935](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10935)


## What was done


* Added class and margin for delete attachment button

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10935`
  * `make fresh`
* Run `make drush-cr`

## How to test


- Log into the site with a test user and select some role that allows you to apply for a grant (for example registered company)
- Start filling in some application for example "Hyvinvoinnin ja terveyden edistämisen yleisavustus" and add some attachment in the last step
- Press "Tallenna keskeneräisenä" and after that go back to edit the attachments

* [x] Make sure the delete attachment button has now margin on top

## Designers review




## Automatic- / Regression tests



[UHF-10935]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10935?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
